### PR TITLE
Fix RTL styles issues

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

RTL styles support

#### Testing instructions
* Go to the page, presented in the screenshot
* Open developer tool and change `dir` attribute of the root `html` element from `ltr` to `rtl`
* Check if the content is appropriately aligned from the right to the left direction.

#### Screenshots
<table>
<tr>
<td>Before:</td><td>After:</td>
</tr>
<tr>
  <td width="50%">
    Screenshot 2022-03-24 at 15 07 13
  </td>
  <td width="50%">
    Screenshot 2022-03-24 at 15 07 27
  </td>
</tr>
<tr>
  <td width="50%">
    Screenshot 2022-03-24 at 16 46 16
  </td>
  <td width="50%">
    Screenshot 2022-03-24 at 16 46 39
  </td>
</tr>

<tr>
  <td width="50%">
    Screenshot 2022-03-24 at 17 06 14
  </td>
  <td width="50%">
    Screenshot 2022-03-24 at 17 06 30
  </td>
</tr>

<tr>
  <td width="50%">
    Screenshot 2022-03-24 at 17 32 24
  </td>
  <td width="50%">
    Screenshot 2022-03-24 at 17 32 36
  </td>
</tr>

<tr>
  <td width="50%">
    Screenshot 2022-03-24 at 17 56 10
  </td>
  <td width="50%">
    Screenshot 2022-03-24 at 17 56 23
  </td>
</tr>

</table>

Related to #57131

### CODE REVIEW

**Comment by bogiii** [comment]
Ready for review☝️

File: client/signup/steps/import/list/style.scss [hunk 1]
```diff
@@ -64,10 +87,10 @@
 		}
 
 		.list__importers-secondary {
-			padding-left: 0;
+			padding: 0;
 
 			@include break-small {
-				padding-left: 72px;
+				padding-inline-start: 72px;
```
Comment (vishnugopal) [inline client/signup/steps/import/list/style.scss:93]: Neat! I didn't know about `padding-inline-start` before!

**Review by vishnugopal — APPROVED** [review]
LGTM! I tested all the pages in both rtl and ltr orientations, sometimes I had to refresh a page for the change to kick-in, but I'm assuming that's because of the manual HTML `dir` property editing.

**Comment by bogiii** [comment]
Yup, the idea is to avoid values such as “left” and “right”, and of course, for some properties is not possible to avoid (float: left and right).

**Pull request merged**